src/Audio_details.md

```diff
@@ -0,0 +1,192 @@
+<style>
+  text {
+    fill: var(--fg);
+    dominant-baseline: middle;
+    font-size: .6em;
+  }
+  #dsg { font-size: 1em; }
+  .centered { text-anchor: middle; }
+  .right    { text-anchor: end; }
+  .tabnum { font-feature-settings: "tnum"; }
+  svg > rect, svg > use {
+    stroke: var(--fg);
+    fill: var(--fg);
+  }
+  svg > path, svg > circle {
+    stroke: var(--fg);
+    fill: none;
+  }
+  .dashed { stroke-dasharray: 2; }
+  .opa50 { opacity: .5; }
+  .fill50 { fill-opacity: .5; }
+  .digital   { stroke: #df0000; fill: #df0000; }
+  .analog    { stroke: #005f98; fill: #005f98; }
+  .misc_data { stroke: #a4269c; fill: #a4269c; }
+  .dac_ena   { stroke: #ffa700; fill: #ffa700; }
+  .ch_active { stroke: #17cc00; fill: #17cc00; }
+  rect.legend { width: 5px; height: 5px; }
+  text.legend { font-size: .7em; }
+  .arrow { stroke-width: 2px; opacity: .35; }
+  .unfilled {
+    fill: none;
+  }
+  .no-stroke {
+    stroke: none;
+  }
+</style>
+
+<figure>
+{{#include imgs/apu_detailed.svg:2:}}
+
+<figcaption>Source: Lior "LIJI32" Halphon</figcaption>
+</figure>
+
+Each of the four "conceptual" channels is composed of a "generation" circuit (designated "channel" in the above diagram), and a [DAC](https://en.wikipedia.org/wiki/Digital-to-analog_converter).
+The digital value produced by the generator, which ranges between $0 and $F (0 and 15), is linearly translated by the DAC into an analog[^digital_analog] value between -1 and 1 (the unit is arbitrary).
+
+The four analog channel outputs are then fed into the mixer[^vin], which selectively adds them (depending on [`NR51`](<#FF25 — NR51: Sound panning>)) into two analog outputs (Left and Right).
+Thus, the analog range of those outputs is 4× that of each channel, -4 to 4.
+
+Then, both of these two get their respective volume scaled, once from [`NR50`](<#FF24 — NR50: Master volume & VIN panning>), and once from the volume knob (if the console has one).
+Note that the former step never mutes a non-silent input, but the latter can.
+
+Each of the two analog outputs then goes through a [high-pass filter](https://en.wikipedia.org/wiki/High-pass_filter) (HPF).
+For short, a HPF constantly tries to "pull" the signal towards analog 0 (neutral); the reason for that is explained further below.
+
+[^digital_analog]:
+To be clear: digital values are discrete and clear-cut; conversely, the analog domain is continuous.
+The former is what computers use, the latter is what the real world is made of.
+
+[^vin]:
+Actually, VIN acts as a 5<sup>th</sup> channel fed into the mixer, whose control bits are in NR50 instead of NR51.
+This was omitted from the diagram for simplicity.
+
+## PCM registers
+
+These two registers, only present in the Game Boy Color and ulterior models, allow reading the output of the generation circuits directly; this is very useful for testing "internal" APU behavior.
+These registers are not documented in any known Nintendo manual.
```

Comment:
What is the purpose of this line? If we specify here should we do it elsewhere through the document as well?

Reply:
Pan Docs currently does not document any such registers [besides these](https://gbdev.io/pandocs/CGB_Registers.html#undocumented-registers), which I am lifting PCMxx from. The line's purpose is to give context, explain why no commercial software uses it, etc. They might be debugging leftovers, for all we know.